### ISSUE TITLE
An Introduction and Offer to Help

### ISSUE THREAD

**Issue by AtlantaJones** [a11yproject/a11yproject.com#89]
So glad to have learned about this project yesterday. I'm a 16-year web developer, currently working for an agency in Atlanta, GA. In my ENTIRE career, both freelance and working for agencies, I'd never dealt with accessibility. It's not that I didn't care, but it's that either a) none of my clients cared, and/or b) they could never afford the extra effort. 

After joining my current agency, I started working on web apps for Wells Fargo bank. And they are fanatical about accessibility. They have to be, or they have to answer to the department of justice. Nothing we do for them can be called complete until it passes accessibility review.

So I was thrust into this whole arena to sink or swim, having almost zero knowledge of how to do it. Over the past 10 months, I've built a couple things for them, most notably this single-page "e-magazine" (https://www.financialadvisorinsights.com), which at first we didn't think could even be done accessibly. 

I've learned a ton, and yet STILL feel like I don't know what I'm doing. For instance, I still have no idea how to use the JAWS screen reader to save my life. One saving grace has been that our main accessibility contact at the client is actually blind himself, and he's been a huge help in determining if this stuff even works. 

So, in short, I don't know how helpful I can be, but I'd really like to contribute to this project. I'll be jotting down some article ideas soon, and maybe I can help with some of the existing ideas list. I could probably easily put something together like doing keyboard accessibility with jQuery, doing "skip to nav" functionality, etc. 

Thanks again to everyone that's made this happen. It's odd that in 2013 this is still such an obstacle, but hopefully things will get easier :)
